onyo/tests/test_usecases.py

```diff
@@ -0,0 +1,205 @@
+import subprocess
+import pytest
+from pathlib import Path
+
+from onyo.lib.onyo import OnyoRepo
+
+
+prefilled_assets = [["warehouse/monitor_dell_PH123.86JZho",
+                     "display: '22.0'\nfzj_inventory: '45FZ18'\n"],
+                    ["warehouse/monitor_dell_NoIdea.S0M3",
+                     "display: '27.0'\n"],
```

Comment:
Just out of curiousity, is there a specific reason, why you end all strings defining the asset contents with a new line, or is it just a style choice? :D

Reply:
A choice for easy enhancement. If it's always `key: value\n` I don't need to think when appending something ;-)